### ISSUE TITLE
Buttons - refined style definition

### ISSUE THREAD

**Issue by RobTobias123** [centrica-engineering/nucleus-docs#387]
As designs have evolved the buttons could do with some rationale and definition.

**Comment by RobTobias123** [comment]
I've defined the buttons as discussed. Point to note is that when we decided to go 16px txt there was an impact on colour ratios, so I've adjusted the pink Values so that where we had 'Pink' that is now 'Pink-500' and where we had 'Magenta' it is now Pink (600). Of course we could just update those 2 values and retain the same names...
 
Pink-500 = HSL 327/60/47 (equiv. Magenta)
Pink (600) = HSL327/60/35 (equiv. Pink)

![Image](https://github.com/centrica-engineering/nucleus-docs/assets/78355810/11479174-b685-419f-af5a-a0222095c5bb)

https://www.figma.com/file/wyuLUEt3qm01mR8E6Wm2Pf/Nucleus-Docs-exploration---engaging-and-interactive?type=design&node-id=466%3A10679&mode=design&t=SpdB59P0efZOhOdg-1
